### DISCUSSION TITLE
Possible issue with sqlserver connection definitions

### DISCUSSION THREAD

**Discussion by jreilly-lukava** [prisma/prisma#10303] [Q&A]
This is a snippet from my schema.prisma
```
datasource db {
  provider = "sqlserver"
  url      = "sqlserver://goblin\\dev;database=josh;integratedSecurity=true;trustServerCertificate=true;"
}
```

If I execute `prisma db pull`, it pulls and creates schema for the database.  However if I try to execute `prisma migrate dev` then I receive the below error message.

```
Error: P1012: error: Error validating: This line is not a valid definition within a datasource.
  -->  schema.prisma:4
   | 
 3 |                 provider = "sqlserver"
 4 |                 url = "sqlserver://goblin\dev;database=josh;integratedSecurity=true;trustServerCertificate=true;"
 5 |             }
   | 
```

I'm at a loss as to why one works but the other does not.

**Comment by janpio** [comment]
With no prior knowledge of SQL Server, what is the quickest way to set up a server this way so it is accessible like that? That would probably be necessary to be able to reproduce this problem.

**Reply by jreilly-lukava** [reply]
I'm not 100% sure what you're asking.  This is just a normal SQL Server Express 2019 installation (installed on Windows Server).  In order to use integrated security I believe you have to be on a Windows client.  However this issue is also manifesting for me if I use a sql user connection string like `DATABASE_URL='sqlserver://goblin\dev;database=josh;user=myUser;password=myPassword;trustServerCertificate=true;'`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
